.github/workflows/on_add_issue.yml

```diff
@@ -0,0 +1,30 @@
+name: Add new issues to the Delivery Assurance project board
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  initialize:
+    name: Initialize new issue
+    runs-on: ubuntu-latest
+    steps:
+      # Rather than use a personal access token to interact with the project, we
+      # can use this GitHub App. There's an API for exchanging app credentials
+      # for a short-term token, and we use that API here.
+      - name: get token
+        uses: tibdex/github-app-token@v1
+        id: app_token
+        with:
+          app_id: ${{ secrets.DELIVERY_ASSURANCE_PROJECT_APP_ID }}
+          installation_id: ${{ secrets.DELIVERY_ASSURANCE_PROJECT_INSTALLATION_ID }}
+          private_key: ${{ secrets.DELIVERY_ASSURANCE_PROJECT_PRIVATE_KEY }}
```

Comment:
[suggestion, non-blocking] [tibdex/github-app-token](https://github.com/tibdex/github-app-token) accepts a `permissions` argument for scoping down permissions. For least privilege / more defensive coding, we could pass in a permission scope like this:

          permissions: >-
            {"issues": "read/write"}

reference: https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
works for me!

Reply:
@Sgtpluck After reviewing the documentation more closely, I think this might be the right syntax: 

      permissions: >-
        {"issues": "write"}

The documentation says "read/write" in its table explaining default permissions, but when I look at the code snippet examples, they just say "write". I think `write` is inclusive of `read` permissions here.

Reply:
done! FYSA i am no longer on the delivery assurance team, so if there are other updates, feel free to make the changes yourself or add a `suggestion` comment for ease of update -- i am just not sure how responsive i will be able to continue to be. thanks!

Reply:
Sounds good -- thanks for the heads up @Sgtpluck! And thanks for working on this!